scipy/linalg/tests/test_lapack.py

```diff
@@ -1654,3 +1654,120 @@ def test_getc2_gesc2():
         else:
             assert_array_almost_equal(desired_cplx.astype(dtype),
                                       x/scale, decimal=4)
+def generate_random_dtype_array(shape, dtype):
```

Comment:
```suggestion


def generate_random_dtype_array(shape, dtype):
```
There should be two blank lines before top-level classes or functions. I've found a style checker becomes really useful for this kind of stuff. I found this using flake8 ``flake8 scipy/linalg/tests/test_linalg.py``.